mirage/config.js

```diff
@@ -85,6 +85,21 @@ export default function() {
 
   this.delete('/user_categories/:id');
 
+  this.get('/skills');
+  this.get('/user_skills');
+
+  // TODO: Make this work when relationships work
```

Comment:
In any case, I would try and remove this custom method and see if the tests run without it. I believe they should.